src/base/bittorrent/session.cpp

```diff
@@ -141,19 +141,55 @@ namespace
         return true;
     }
 
+#ifdef Q_OS_WIN
+    // Remove this after 4.2.5 && if at least one month has passed from v4.2.3
```

Comment:
Maybe we should announce this known problem on our website, the NEWS page and discourage people from using it (v4.2.2)?

Reply:
I intend to mention it in the v4.2.3 release entry.

Reply:
It wouldn't hurt to prepend this with TODO. It's a good idea in general especially considering a lot of modern IDEs will highlight TODOs for you.